test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,73 @@
+require "test_helper"
+
+describe UsersController do
+
+  describe "index" do
+    it "can get the index path" do
+      # Act
+      get users_path
+      # Assert
+      must_respond_with :success
+    end
+  end
+
+  describe "new" do
+    it "can get the new user" do
+
+      # Act
+      get new_user_path
+
+      # Assert
+      must_respond_with :success
+    end
+  end
+
+  describe "show" do
+    it "can get a valid user" do
+          # Act
+          get user_path(first.id)
```

Comment:
Missing The model class

```suggestion
      get user_path(User.first.id)
```